Resources/Prototypes/Catalog/Fills/Items/gas_tanks.yml

```diff
@@ -91,6 +91,28 @@
           - 22.6293856 # nitrogen
         temperature: 293.15
 
+- type: entity
+  id: NitrousOxideTankFilled
+  parent: NitrousOxideTank
+  suffix: Filled
+  name: nitrous oxide tank
+  components:
+    - type: GasTank
+      outputPressure: 101.325
+      air:
+        volume: 70
+        moles:
+          - 4 # oxygen
+          - 16.5 # nitrogen
+          - 0 # CO2
+          - 0 # Plasma
+          - 0 # Tritium
+          - 0 #  Water vapor
+          - 0 # Miasma
+          - 2 # N2O
```

Comment:
shouldnt this mostly be o2/n2o and not n2. is it tested that they can still survive but just fall asleep?

Reply:
Yeah I tested everything. I guess I'll just add all of the nitrogen to oxygen